localstack/config.py

```diff
@@ -277,23 +352,30 @@ def in_docker():
 # name of the host under which the LocalStack services are available
 LOCALSTACK_HOSTNAME = os.environ.get("LOCALSTACK_HOSTNAME", "").strip() or LOCALHOST
 
-# directory for persisting data
+# directory for persisting data (TODO: deprecated, simply use PERSISTENCE=1)
 DATA_DIR = os.environ.get("DATA_DIR", "").strip()
 
+# whether localstack should persist service state across localstack runs
+PERSISTENCE = is_env_true("PERSISTENCE")
+
+# whether to clear config.dirs.tmp on startup and shutdown
+CLEAR_TMP_FOLDER = is_env_not_false("CLEAR_TMP_FOLDER")
+
 # folder for temporary files and data
 TMP_FOLDER = os.path.join(tempfile.gettempdir(), "localstack")
 
+# this is exclusively for the CLI to configure the container mount into /var/lib/localstack
+VOLUME_DIR = os.environ.get("LOCALSTACK_VOLUME_DIR") or TMP_FOLDER
```

Comment:
nit: Can be helpful to accept `" "` (an empty string of only whitespaces) also as an empty value. (e.g., if users are using some configuration mechanism that requires them to specify a value, but it should be considered empty). Arguably a bit of a corner case, but has proven useful for other config variables a few times in the past.. 👍  

```suggestion
VOLUME_DIR = os.environ.get("LOCALSTACK_VOLUME_DIR", "").strip() or TMP_FOLDER
```